modules/core/src/smithy4s/Errorable.scala

```diff
@@ -16,8 +16,14 @@
 
 package smithy4s
 
-trait Errorable[E] {
-  def error: schema.Schema.UnionSchema[E]
+trait Errorable[E] extends ShapeTag[E] {
```

Comment:
I think I'm gonna move this in the `schema` package in a subsequent PR, and rename to `ErrorSchema`